src/server/routes/contract/write/write.ts

```diff
@@ -26,19 +26,35 @@ const writeRequestBodySchema = Type.Object({
   functionName: Type.String({
     description: "The function to call on the contract",
   }),
-  args: Type.Array(
-    Type.Union([
-      Type.String({
-        description: "The arguments to call on the function",
-      }),
-      Type.Tuple([Type.String(), Type.String()]),
-      Type.Object({}),
-      Type.Array(Type.Any()),
-      Type.Any(),
-    ]),
-  ),
+  args: Type.Array(Type.Any(), {
+    description: "The arguments to call on the function",
+  }),
   ...txOverridesWithValueSchema.properties,
-  abi: Type.Optional(Type.Array(abiSchema)),
+  abi: Type.Optional(Type.Array(Type.Object({
```

Comment:
Is this different than abiSchema below (in `src/server/schemas/contract/index.ts`)? Hard to tell but it looks the same at a glance.

Reply:
Looks the same as I walk through it.